src/test/java/JavaClass7/JavaHomeWork7.java

```diff
@@ -2,48 +2,178 @@
+        } else if (CarModel4.equalsIgnoreCase("Tesla") || CarModel4.equalsIgnoreCase("Kia EV9")) {
+            System.out.println("The Vehicle type is : ‘Electric’ ");
+        } else if (CarModel4.equals("Lexus RX350") || CarModel4.equals("Toyota RAV4")) {
+            System.out.println("The Vehicle type is : ‘SUV’ ");
+        } else if (CarModel4.equalsIgnoreCase("Porsche") || CarModel4.equalsIgnoreCase("Ferrari")) {
+            System.out.println("The Vehicle type is : ‘Sports Car’ ");
+        } else {
+            System.out.println("Unknown Vehicle type ");
+        }
+
+
+        if (CarModel5.equals("Honda") || CarModel5.equals("Toyota")) {
+            System.out.println("The Vehicle type is : ‘Sedan’ ");
+        } else if (CarModel5.equalsIgnoreCase("Tesla") || CarModel5.equalsIgnoreCase("Kia EV9")) {
+            System.out.println("The Vehicle type is : ‘Electric’ ");
+        } else if (CarModel5.equalsIgnoreCase("Lexus RX350") || CarModel5.equalsIgnoreCase("Toyota RAV4")) {
+            System.out.println("The Vehicle type is : ‘SUV’ ");
+        } else if (CarModel5.equalsIgnoreCase("Porsche") || CarModel5.equalsIgnoreCase("Ferrari")) {
+            System.out.println("The Vehicle type is : ‘Sports Car’ ");
+        } else {
+            System.out.println("Unknown Vehicle type ");
+        }
+
+        if (CarModel6.equals("Honda") || CarModel6.equals("Toyota")) {
+            System.out.println("The Vehicle type is : ‘Sedan’ ");
+        } else if (CarModel6.equalsIgnoreCase("Tesla") || CarModel6.equalsIgnoreCase("Kia EV9")) {
+            System.out.println("The Vehicle type is : ‘Electric’ ");
+        } else if (CarModel6.equalsIgnoreCase("Lexus RX350") || CarModel6.equalsIgnoreCase("Toyota RAV4")) {
+            System.out.println("The Vehicle type is : ‘SUV’ ");
+        } else if (CarModel6.equalsIgnoreCase("Porsche") || CarModel6.equalsIgnoreCase("Ferrari")) {
+            System.out.println("The Vehicle type is : ‘Sports Car’ ");
+        } else {
+            System.out.println("Unknown Vehicle type ");
+        }
+
+        if (CarModel7.equals("Honda") || CarModel7.equals("Toyota")) {
+            System.out.println("The Vehicle type is : ‘Sedan’ ");
+        } else if (CarModel7.equalsIgnoreCase("Tesla") || CarModel7.equalsIgnoreCase("Kia EV9")) {
+            System.out.println("The Vehicle type is : ‘Electric’ ");
+        } else if (CarModel7.equalsIgnoreCase("Lexus RX350") || CarModel7.equalsIgnoreCase("Toyota RAV4")) {
+            System.out.println("The Vehicle type is : ‘SUV’ ");
+        } else if (CarModel7.equalsIgnoreCase("Porsche") || CarModel7.equalsIgnoreCase("Ferrari")) {
+            System.out.println("The Vehicle type is : ‘Sports Car’ ");
+        } else {
+            System.out.println("Unknown Vehicle type ");
+        }
+
+        int GradeA =100;
+        int GradeB = 89;
+        int GradeC = 79;
+        int GradeD = 70;
+
+
+        if (GradeA <= 100 ) {
```

Comment:
For each **if** statement you have to apply comparison and logical operator (please refer to JavaClass3 line 130-150)
Hint : Select logical operator, where left and right statement returns "true"; 
Your check should be able to handle ranges Ex: from 90 -100 in one line of code; from 80-90 in second if statement etc...